Formula/libdivsufsort.rb

```diff
@@ -0,0 +1,70 @@
+class Libdivsufsort < Formula
+  desc "Lightweight suffix-sorting library"
+  homepage "https://github.com/y-256/libdivsufsort"
+  url "https://github.com/y-256/libdivsufsort/archive/refs/tags/2.0.1.tar.gz"
+  sha256 "9164cb6044dcb6e430555721e3318d5a8f38871c2da9fd9256665746a69351e0"
+  license "MIT"
+  head "https://github.com/y-256/libdivsufsort.git", branch: "master"
+
+  depends_on "cmake" => :build
+
+  def install
+    args = %w[
+      -DBUILD_EXAMPLES=OFF
+      -DBUILD_DIVSUFSORT64=ON
+      -DUSE_OPENMP=ON
```

Comment:
Does setting this make sense without any kind of OpenMP dependency?

Reply:
As far as I can tell, it does seem to work. On Linux, it uses system GCC's libgomp. On macOS, it seems to add `-openmp` to `CFLAGS`.

Maybe it works for me on macOS because I already have `libomp` installed? Although I don't see any linkage from this to `libomp`.

Reply:
Locally, I see it perform this test, regardless of whether or not I add `libomp` as a dependency. If I do add `libomp` as a dependency, there's no linkage.

```
-- Performing Test HAVE_UNKNOWN_WALL
-- Performing Test HAVE_UNKNOWN_WALL - Success
-- Performing Test HAVE_UNKNOWN_FOMIT_FRAME_POINTER
-- Performing Test HAVE_UNKNOWN_FOMIT_FRAME_POINTER - Success
-- Performing Test HAVE_UNKNOWN_FOPENMP
-- Performing Test HAVE_UNKNOWN_FOPENMP - Failed
-- Performing Test HAVE_UNKNOWN_OPENMP
-- Performing Test HAVE_UNKNOWN_OPENMP - Success
-- Performing Test HAVE_UNKNOWN_OMP
-- Performing Test HAVE_UNKNOWN_OMP - Success
```

Reply:
I've just removed the flag to enable OpenMP for now, to keep things simpler.